dist/pinny.js

```diff
@@ -467,6 +470,26 @@
             return $(document.activeElement);
         },
 
+        _isWithin: function($element, selector) {
+            return ($element.closest('.' + selector).length) ? true : false;
+        },
+
+        /*
+        * In iOS, blur keyboard on orientationchange to avoid position fixed
+        * input elements from shifting
+        */
+        _fixIOSInputJumpOnFocus: function() {
+            var plugin = this;
+            if ($.os.ios) {
+                $(window).on('orientationchange', function(e) {
+                    var $focusedElement = plugin._activeElement();
```

Comment:
I know this is MAJORLY nit picky, but I think code is *way* more readable when, as a rule, you separate `var` statements from other code. What do you think?